com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumePerSceneData.cs

```diff
@@ -77,7 +77,16 @@ internal void InvalidateAllAssets()
             foreach (var asset in assets.Values)
             {
                 if (asset != null)
+                {
                     ProbeReferenceVolume.instance.AddPendingAssetRemoval(asset);
+#if UNITY_EDITOR
+                    if (AssetDatabase.TryGetGUIDAndLocalFileIdentifier(asset, out string guid, out long file))
```

Comment:
why not use AssetDatabase.GetAssetPath ?

Reply:
I was actually testing something else for which I needed also the GUID, now indeed that one makes more sense; will try tomorrow.